bitshuffle/bitshuffle.py

```diff
@@ -190,7 +190,7 @@ def main():
     args = parser.parse_args()
 
     # Checks if no parameters were passed
-    if len(sys.argv[1:]) == 0:
+    if sys.argv[1:] == 0:
```

Comment:
This does NOT do what is intended, see [hanging build]

Reply:
> Actually, I could have left it with len() in retrospect, as it would do the same thing.

Okay, maybe not lol.